src/wallet/rpc/transactions.cpp

```diff
@@ -642,7 +642,7 @@ RPCHelpMan listsinceblock()
         const CWalletTx& tx = pairWtx.second;
 
         if (depth == -1 || abs(wallet.GetTxDepthInMainChain(tx)) < depth) {
-            ListTransactions(wallet, tx, 0, true, transactions, filter, nullptr /* filter_label */, /*include_change=*/include_change);
+            ListTransactions(wallet, tx, 0, true, transactions, filter, /*filter_label=*/filter_label, /*include_change=*/include_change);
```

Comment:
115a145c70fc5f7209a41523e3c0fbec7e3732ce won't compile because `filter_label` is declared in the next commit.
 Use `std::nullopt` here.